packages/hash/api/src/graphql/typeDefs/knowledge/user.typedef.ts

```diff
@@ -1,6 +1,6 @@
 import { gql } from "apollo-server-express";
 
-export const userTypedef = gql`
+export const persistedUserTypedef = gql`
```

Comment:
@Alfred-Mountfield is getting rid of `persisted` in a PR today(?) so might be worth calling this whatever it would be in the new world.

Reply:
To my understanding Alfie is getting rid of `persisted` in the `"@hashintel/hash-graph-client"` package today, and that removing the uses of `persisted` will happen in follow-up (likely by myself when I am able to start on [this task](https://app.asana.com/0/1201095311341924/1203227079758117/f)). I think this should be merged to help ensure that we are renaming these variables consistently in the same PR.